internal/pkg/config/profiles_test.go

```diff
@@ -182,3 +183,87 @@ func TestImportProfile(t *testing.T) {
 		})
 	}
 }
+
+func TestExportProfile(t *testing.T) {
+	// Create directory where the export configs should be stored
+	testDir, err := os.MkdirTemp(os.TempDir(), "stackit-cli-test")
+	if err != nil {
+		t.Fatal(err)
+	}
+	t.Cleanup(func() {
+		func(path string) {
+			err := os.RemoveAll(path)
+			if err != nil {
+				fmt.Printf("could not clean up temp dir: %v\n", err)
+			}
+		}(testDir)
+	})
+
+	// Create prerequisite profile
+	p := print.NewPrinter()
+	profileName := "export-profile-test"
+	err = CreateProfile(p, profileName, true, true)
+	if err != nil {
+		t.Fatalf("could not create prerequisite profile, %v", err)
+	}
```

Comment:
This would allow us even to have more control and test the case where the profile doesn't exist